pkg/health/pods.go

```diff
@@ -134,7 +134,7 @@ func PodsUnhealthy(client k8sclient.Client, options *k8sclient.ListOptions) (*[]
 	return &unhealthy, nil
 }
 
-func unhealthyStatusConditions(pod corev1.Pod) bool {
+func UnhealthyStatusConditions(pod corev1.Pod) bool {
```

Comment:
please drop the change, look above comment for reasons

Reply:
Done